sandbox/src/mocks/cancelReferral/responses/CancelledReferralDeferredIntendPrivateWithComment.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
Raise with BAs.

Reply:
This was changed to a scenario "Cancelled referral with existing review outcome", which includes a resolved deferral.